underlay/src/main/resources/config/datamapping/aouRT/entitygroup/conditionPerson/entityGroup.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "conditionPerson",
```

Comment:
Name it `conditionSnomedPerson` since it's the relationship between SNOMED conditions and person